docs/user-guide/api-mediation/configuration-connection-limits.md

```diff
@@ -3,10 +3,37 @@
 :::info Role: system programmer
 :::
 
+## TCP/IP Connection Limits
+
 By default, the API Gateway accepts up to 100 concurrent connections per route, and 1000 total concurrent connections. Any further concurrent requests are queued until the completion of an existing request. The API Gateway is built on top of Apache HTTP components that require these two connection limits for concurrent requests. 
 
 Use the following procedure to change the number of concurrent connections.
 
 1. Open the file `zowe.yaml`.
-2. Find or add the property `components.gateway.server.maxConnectionsPerRoute` and set the value to an appropriate positive integer.
-3. Find or add the property `components.gateway.server.maxTotalConnections` and set the value to an appropriate positive integer.
\ No newline at end of file
+2. Find or add the property `zowe.components.gateway.server.maxConnectionsPerRoute` and set the value to an appropriate positive integer.
+3. Find or add the property `zowe.components.gateway.server.maxTotalConnections` and set the value to an appropriate positive integer.
+
+## Websocket Limits
+
+The API Mediation Layer supports Websocket connections. It's possible to configure the limits around timeouts, all the values are in miliseconds. This may be relevant if you see problems for example around the usage of TN32790 terminal in Virtual Desktop. 
+
+```
+zowe:
+  components:
+    gateway:
+      apiml:
+        server:
+          websocket:
+            connectTimeout: 15000
+            stopTimeout: 30000
+            asyncWriteTimeout: 60000
+            maxIdleTimeout:3600000
+```
+
+Use the following procedure to change the limits:
+
+1. Open the file `zowe.yaml`.
+2. Find or add the property `zowe.components.gateway.apiml.server.websocket.connectTimeout` and set the value to an appropriate positive integer. This timeout limits how long the API Gateway waits untill it drops connection if it can't reach the target server.
+3. Find or add the property `zowe.components.gateway.apiml.server.websocket.` and set the value to an appropriate positive integer. This timeout handles how long the API Gateway waits before it fails on stop message for teh Websocket connection
+4. Find or add the property `zowe.components.gateway.apiml.server.websocket.` and set the value to an appropriate positive integer. This timeout handles how long it takes before the server fails with unsuccessful response when trying to write message to the Websocket connection. 
+5. Find or add the property `zowe.components.gateway.apiml.server.websocket.` and set the value to an appropriate positive integer. This timeout handles how long the Websocket connection remains open if there is no communication happening over the open connection. 
```

Comment:
the properties' prefix is `zowe.components.gateway.server`, without the apiml part

Reply:
Changed everywhere.